### PR TITLE
Uno, Sensenet Updated, ArchivesSpace Added 

### LINKED ISSUE #734: Archives space Hacktoberfest
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : ArchivesSpace

Q. Where have you seen the announcement (URL)?  
A. : https://archivesspace.org/archives/6563

Not sure about this one...

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by souravroy-test**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Closes #734

### CODE REVIEW

**Comment by souravroy-test** [comment]
@aslafy-z can you please check this out?

File: data.json [hunk 1]
```diff
@@ -70,6 +70,15 @@
...
     "dateAdded": "2018-10-24T11:45:00.000Z",
     "tags": ["clothing", "hacktoberfest", "expired", "stickers"]
+  },
+    {
+    "name": "ArchivesSpace",
+    "difficulty": "medium",
+    "description": "Create PRs to <a href='https://github.com/archivesspace/archivesspace'>ArchivesSpace GitHub repos</a> and the Top COntribuitor will get a ArchivesSpace bag",
+    "reference": "https://community.aras.com/b/english/posts/join-aras-labs-hacktoberfest-2018",
+    "image": "https://archivesspace.org/wp-content/uploads/2016/05/aspacelogo.png",
+    "dateAdded": "2020-04-10T15:21:57.000Z",
```
Comment (aslafy-z) [inline data.json:80]: These dates should be formated as YYYY-MM-DD

File: data.json [hunk 2]
```diff
@@ -641,11 +650,11 @@
   {
     "name": "Uno",
     "difficulty": "easy",
-    "description": "Register for the Hacktoberfest event. Share your contact details. Make a pull-request to improve the projects code or documentation.",
-    "reference": "https://platform.uno/uno-is-joining-hacktoberfest-2019/",
+    "description": "Register for the Hacktoberfest event. Make a valid PR and fill up a <a href='https://www.surveymonkey.com/r/FNDG269'>form</a>",
```
Comment (aslafy-z) [inline data.json:653]: As stated in #754, we won't list this opportunity this year. Please reset you're changes

Comment (souravroy-test) [reply data.json:653]: Ok, removing Uno & Correcting date format

File: data.json (hunk 1 above)
Comment (souravroy-test) [reply data.json]: Done!

**Pull request closed without merging**